src/content/reference/manifest.mdx

```diff
@@ -120,6 +120,14 @@ A list of commands to deploy your development environment.
 
 It's usually a combination of `helm`, `kubectl`, and `okteto` commands.
 
+#### Deploy a `docker-compose` file:
+
+```yaml
+deploy:
```

Comment:
I find thus very confusing, since we say in other places that all you need is the compose file. What's the benefit of having this in the deploy vs not having an okteto.yml file?

Reply:
You might want to define dev containers, add `dependencies` or deploy k8s resourcces

Reply:
It's part of what we accept in the schema. We could remove this to encourage users to use only a `docker-compose` or go directly to the advanced features (multiple composes, dependencies, k8s resources, only deploy subset...). But there'll be times when you want to create an `okteto manifest` to specify another section (like `dev`) and that's when this notation is useful

Reply:
I agree is useful, but we should tell our users why is useful, and make it clear that this is not required. 

I think that the use case of "customize the dev section" makes a lot of sense. The "deploy dependencies" or "k8s resources" sounds like a more advanced use case to me. I think that should be documented somewhere else.

Reply:
@rberrelleza I changed it to redirect users with only a docker-compose to its corresponding guide.

If someone is going through the `manifest schema reference`, I think they should see all the available options.
If not, I think we could split it into 2:
- a specification of the manifest schema
- a simpler user guide